qor-example amazon pay config bug

while trying to run the example app the following error occurs: 
`# github.com/qor/qor-example/config
config/config.go:94:12: cannot use amazonpay.New(&amazonpay.Config literal) (type amazonpay.AmazonPayService) as type *amazonpay.AmazonPay in assignment: need type assertion`

--- a/config/config.go
+++ b/config/config.go
@@ -66,7 +66,7 @@ var (
 	Root           = os.Getenv("GOPATH") + "/src/github.com/qor/qor-example"
 	Mailer         *mailer.Mailer
 	Render         = render.New()
-	AmazonPay      *amazonpay.AmazonPay
+	AmazonPay      amazonpay.AmazonPayService
 	PaymentGateway gomerchant.PaymentGateway
 	RedirectBack   = redirect_back.New(&redirect_back.Config{
 		SessionManager:  manager.SessionManager,

